Addition of User Story 4.4

eric-schuh opened the pull request:
First draft of User Story 4.4 Primary Education Impact Bond

eric-schuh committed:
Addition of User Story 4.4

First draft of User Story 4.4 Primary Education Impact Bond

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
 #### Context
+Ruari is the director of a large education charity which focuses on building and updating infrastructure in low income schools in California. Recently his organization has been investigating a problem they have noticed where a large percentage of schools in the lower income counties have not had their lab equipment updated in close to 30 years, which has lead to, according to a set of studies, an ...
+
+In an attempt to solve this issue, Ruari’s organization setups what they are calling the New Labs Initiative Impact Bond, and have been able to get the promises of two other local charity groups in Los Angeles, where they will be piloting the program, which collectively have pledged a total of $10 million to the outcome fund. The three charities have collectively set outcome criteria as a functio ...
```
sp. sets up

ig-shaun replied:
In defining outcome states to be achieved, it would be preferable to measure the academic results of students. This allows for a claim to be made that x number of students achieve y academic performance (e.g. a threshold score on independent testing - like SAT, but specifically for science). If there is a known baseline for the number (or percentage) of students who would likely pass, this allows for targets to be set for expected future state of pass numbers or rate, with investment into the intervention.
We want to be able to tokenise these academic outcomes which would conventionally have an analogue representation, such as an academic certificate of achievement, based on verified claims. 
Ideally one token represents a verifiable credential for one student who has achieved the required threshold score for a certificate of a certain grade to have been issued. The token is issued by a trusted issuer (such as the examination authority, or an issuer authorised to act on behalf of the authority).
The outcome payers are then effectively buying these outcomes, conditional to a sufficient absolute number, or percentage relative to the denominator number of students, having been issued.

These NFTs have the requirement that they are linked to independently evaluated and verified claims, with supporting evidence. The tokens must not compromise the privacy of the students (or other private agents and agencies).

ig-shaun replied:
'commitment' rather than promises - generally these arrangements need to resolve to contractual obligations

ig-shaun replied:
It should be made explicit that the outcome funders are prepared to purchase outcomes. These outcomes are economically valuable and have willing 'buyers'.

As an aside, the outcomes could also be associated with rights - such as admission rights, or scholarships.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+User needs here are broken down for each of the three users presented in the above story: Ellenor, Vicki and Stephen. 
+
+__Outcome Funder__
+* Configuration of a clear set of criteria on which to base the outcome payments
```
Define measurable outcome states which can be independently verified

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
+In an attempt to solve this issue, Ruari’s organization setups what they are calling the New Labs Initiative Impact Bond, and have been able to get the promises of two other local charity groups in Los Angeles, where they will be piloting the program, which collectively have pledged a total of $10 million to the outcome fund. The three charities have collectively set outcome criteria as a functio ...
+
+Once the outcome funding has been pledged, Ruari approaches several companies in the local Los Angeles region and gets buy-in from four which agree to start upgrading the schools per the bond specifications. These companies take on the upfront capital investment costs of the upgrades which, when completed at a particular location, are evaluated by either a representative of the specific school’s  ...
```
I would simplify this for the purpose of this user story, to define the outcome states as being academic achievements, which can be demonstrated by independent examinations. This allows for a more granular definition of the units of value (academic certificates of achievement - above a threshold performance level), which can be tokenised. This also has an existing analogue resource, in the form of academic certificates of achievement.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
+Once the outcome funding has been pledged, Ruari approaches several companies in the local Los Angeles region and gets buy-in from four which agree to start upgrading the schools per the bond specifications. These companies take on the upfront capital investment costs of the upgrades which, when completed at a particular location, are evaluated by either a representative of the specific school’s  ...
+
+The verifiable claims submitted by Company A prove that they upgraded labs which will service 10,000 students and that this covers 35% of the students in East Los Angeles County. They receive the specified payout from the outcome fund based on this claim bundle for both the site specific and meta-level metrics they achieved. This project continues over the next 5 years with various companies join ...
```
Generally outcome payers would be more incentivised to pay for outcomes that can be valued (eg a student having passed their academic assessment, which leads to further opportunities with societal and economic value) vs outputs, such as the upgraded state of the facility.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+__Outcome Funder__
+* Configuration of a clear set of criteria on which to base the outcome payments
+* Assurance that reported outcomes are properly validated by a clear and mutually agreed upon validation method
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
```
We should separate the role of investor vs implementor, as this reduces conflicts of interest. Impact bonds would typically have capital market investors. In this user story, the companies doing the lab refurbs would more likely borrow the capital from banks against performance-based contracts for future outcome payments (which usually include at least a coupon value premium and often also include an additional incentive payment for performance).

The way bonds are structured, the capital investor/s take the financial risks on their capital principal for whether the outcome payment is released or not. The implementor may have at-risk the performance bonus.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+* Assurance that reported outcomes are properly validated by a clear and mutually agreed upon validation method
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
+
+__Investing Companies__
```
As per comments above - separate the role of Capital Providers (Investors) vs Implementers

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+* Mechanism to make the outcome payments as the various companies who are fronting the upfront costs reach desired outcome levels
+
+__Investing Companies__
+* Clear criteria for receiving outcome payments
```
Clearly-defined outcome states that trigger outcome payments

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+
+__Investing Companies__
+* Clear criteria for receiving outcome payments
+* Clear return on investment when the criteria for an outcome payment is met
```
Financial returns on investment, which could include a coupon premium, performance bonus (though this is often passed through to the implementers), share of Impact Tokens generated - which is interesting in contexts where these tokens are tradable because there is demand and liquidity, or specific rights that can be accessed through owning the tokens (eg data access)

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+__Investing Companies__
+* Clear criteria for receiving outcome payments
+* Clear return on investment when the criteria for an outcome payment is met
+* Mechanism to receive the outcome payments once the company has reached the specified criteria
```
For disambiguation 'company' could be described as 'implementer'

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+* Mechanism to receive the outcome payments once the company has reached the specified criteria
 
 #### Unique Challenges
+The metadata in this user story comes in when a company is reporting the outcome. This would be:
```
Not a strong class of metadata.
Rather use Academic Certificates of Achievement, with the primary challenge being privacy

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+* Where the serviced school is located
+* How many students will the upgraded labs service
+
+Given that the outcome funders want assurance that the company has properly completed the task for the outcome payment this means that the NFT generated from a token representing a set outcome claims will need to be:
```
Claims (of academic achievement) are independently evaluated (through independent academic testing) and verified by a trusted authority (such as the academic testing authority).
Claims which are verified as true claims and evaluated as positive (meeting the criteria for approval) provide the basis for issuing Verifiable Credentials (the digital equivalent of certificate of achievement or academic qualification). NFTs are minted to represent these academic outcome states.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+Given that the outcome funders want assurance that the company has properly completed the task for the outcome payment this means that the NFT generated from a token representing a set outcome claims will need to be:
+__Metadata Requirements:__ Mintable, Burnable, Ownable, Transferable, Lockable, Immutable, Composible
+
+In addition there is the interesting problem of how the outcome claims are verified. Likely this project would need buy-in from at least a set of school administrators who are willing to participate in the verification and feedback mechanisms that the outcome funders have decided they will need to have confidence in funding this project. 
```
Rather describe an independent academic evaluation mechanism

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+In addition there is the interesting problem of how the outcome claims are verified. Likely this project would need buy-in from at least a set of school administrators who are willing to participate in the verification and feedback mechanisms that the outcome funders have decided they will need to have confidence in funding this project. 
 
 #### Distinctive Requirements
+Companies taking part in this project will want to be able to, in various circumstances, submit both outcome claims for a particular instance, i.e. one school has been serviced, while also having the ability to submit a group of claims at the same time. This will allow for the project to have flexible outcome payments which should enable companies both small, who may need to receive the ROI as so ...
```
This scenario of 'flexible outcome payments' would not be typical for a bond instrument.

Distinctive requirements would more likely be

- Standardisation of measures
- Standardisation of evaluation methodologies
- Controlling access to sensitive information, whilst providing assurance that claims and verifications of these claims are valid
- Sufficient information in the claim resources to assess the nature of the outcome and/or beneficiary

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+#### NFT Metadata Layer Model
+| Layer |  |
+|-----|------|
+| **8. Operationalization** | Holder of the verified claim NFT submits a transaction to the bond chain and receive the associated payment |
```
'bond mechanism' (not 'chain')

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+| **7. Assertions** | Verifiable claims are assertions by witnesses about the underlying impact claim, which constitute evidence for oracles. | 
+| **6. Extensions &    Restrictions to Rights** | N/A | 
+| **5. Instantiation** | A particular token for a particular set of claims asserting the creation of a particular outcome. |  
+| **4. Embodied Rights** | Redeemable for a particular outcome payment. |
```
May include other rights such as right of admission to next academic level, right to receive bursary or subsidy funding, ...

ig-shaun replied:
Or transferrable.
Another unique requirement is that it must be possible to 'retire' or deactivate these tokens after a certain period, as they are only valid for trade and payment within the timeframe of the student's academic period

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+|-----|------|
+| **8. Operationalization** | Holder of the verified claim NFT submits a transaction to the bond chain and receive the associated payment |
+| **7. Assertions** | Verifiable claims are assertions by witnesses about the underlying impact claim, which constitute evidence for oracles. | 
+| **6. Extensions &    Restrictions to Rights** | N/A | 
```
May be applicable based on the attributes of the beneficiary, for instance, age, gender, socio-economic status, ... which may be based on verifiable credentials

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+| Layer |  |
+|-----|------|
+| **8. Operationalization** | Holder of the verified claim NFT submits a transaction to the bond chain and receive the associated payment |
+| **7. Assertions** | Verifiable claims are assertions by witnesses about the underlying impact claim, which constitute evidence for oracles. | 
```
Assertions are not necessarily made by 'witnesses'. For instance, a beneficiary may make their own assertion that they are 'ready to pass the grade'. Independent academic assessment tests this assertion and a qualified authority (the examination body) makes an assertion of the academic performance grade which has been achieved, based on test results

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+| **6. Extensions &    Restrictions to Rights** | N/A | 
+| **5. Instantiation** | A particular token for a particular set of claims asserting the creation of a particular outcome. |  
+| **4. Embodied Rights** | Redeemable for a particular outcome payment. |
+| **3. Token Logic** | The NFT created by the verification chain oracle is a cryptographically transferable resource. | 
```
Needs modification according to the changes in what is being measured, valued and traded. With validity periods, etc.

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+| **5. Instantiation** | A particular token for a particular set of claims asserting the creation of a particular outcome. |  
+| **4. Embodied Rights** | Redeemable for a particular outcome payment. |
+| **3. Token Logic** | The NFT created by the verification chain oracle is a cryptographically transferable resource. | 
+| **2. Interchain** | The verification chain supports oracles who can produce NFTs which, themselves, can trigger payment transfers on the cryptocurrency chain. | 
```
Impact Tokens are transferrable across chains and metadata can be accessed across chains. For instance, the metadata may be written to a confidential storage chain, whereas the NFT is minted on another chain

ig-shaun commented on NFT-RFC-002-use-cases.md:
```diff
@@ -411,12 +411,52 @@ Uncertainty: “How sure are we about the monitoring and quantification methodol
+| **4. Embodied Rights** | Redeemable for a particular outcome payment. |
+| **3. Token Logic** | The NFT created by the verification chain oracle is a cryptographically transferable resource. | 
+| **2. Interchain** | The verification chain supports oracles who can produce NFTs which, themselves, can trigger payment transfers on the cryptocurrency chain. | 
+| **1. State Machine** | Cryptocurrency chain for Outcome Funders, Verification Chain for collecting and verifying original assertions of root outcomes. |
```
Ledger records token identifiers, balances and transaction history.
Stateful database stores metadata about claims, credentials and their state.

ig-shaun reviewed (changes requested):
General comment - it is probably more correct to define this as a Secondary School Education intervention (primary school education is typically early grades).

We want to capture the concept of (Impact Token) NFTs capturing discretely measured outcome states, which have analogue equivalents such as Academic Certificates which can be digitally represented as Verifiable Credentials.
Unique requirements to emphasis here are the need to assure privacy and the need to have a validity period for certain specified types of transactions which can be performed with this tokens.

eric-schuh committed:
Update NFT-RFC-002-use-cases.md

updates to user story 4.4, rewrite of the context and unique challenges sections. Updates to Distinctive Requirements and the Architecture Model to come
